### ISSUE TITLE
Design: Button updates (v2)

### ISSUE THREAD

**Issue by RavinaSamaroo** [cds-snc/design-gc-conception#1242]
## User story

As a GC Design System user...
I want the GC Design System to include the Supertask button...
so that the GCDS Button can be aligned with the standards found on design.canada.ca

Button redesign includes:

Adding start/supertask button
adding small buttons (view decision doc)
changing disabled state from grey to 50% opacity

### Done when

- [x] Update buttons in [EN library](https://www.figma.com/design/mh2maMG2NBtk41k1O1UGHV/branch/OPMe9xHljUSgMuwy8CWOa9/GC-Design-System?node-id=817-4607&node-type=&t=K4Pk1eRHU6eHYIXH-0)
- [x] Update buttons in [FR library](https://www.figma.com/design/o4SguSZdar2CCFzSkWNrmB/branch/57H9vu6WJg3wx4LngDu9R3/Syst%C3%A8me-de-design-GC?node-id=8-2054&node-type=&t=Gw0G8niNT3WGrdIo-0)
- [x] Update component archive [Figma file](https://www.figma.com/design/4wF8Kr9o2sGTQjaOpY7qcZ/Component-archive-%2B-decision-record?node-id=2693-12035&node-type=section&t=zODfuCfPNDCKTYXw-0)

**Comment by ClementineHahn** [comment]
Presented in sprint review Nov 14th, Poster cant be updated until review is finalized . Will be next step

**Comment by ClementineHahn** [comment]
@RavinaSamaroo , can you add the links of the work you presented yesterday ? Thank you

**Comment by SmartMouthWords** [comment]
Do we need another card to capture the content/translation and then another for the publishing?

**Comment by daine** [comment]
@SmartMouthWords yes, and we need a card for the development changes as well (I'll check and see if one already exists)

**Comment by SmartMouthWords** [comment]
Will changes to the button disabled state address this [issue](https://github.com/cds-snc/gcds-components/issues/720#issuecomment-2601512709)?

**Comment by SmartMouthWords** [comment]
May require a new branch to ensure all updates are applied. @nmakuch Analyzing

**Comment by ClementineHahn** [comment]
Blocker :need to make a call for the decision doc for the buttons . It need to be documented in the decision doc @nmakuch to add the link for reference

**Comment by SmartMouthWords** [comment]
Blocked. Waiting on DTO review of guidance and new start name for supertask.

**Comment by daine** [comment]
Hi! The following tasks are more in line with release and publishing:
- Update EN poster
- Update FR poster
- Draft of [EN guidance](https://docs.google.com/document/d/1IGYNg2R6j5F72uVkXb5oP9Xj78Zx-CPqB6UzH6quTUM/edit?tab=t.0#heading=h.tbswke9yeyw2) with alt text for any images
- Anatomy image created and converted for publishing

Flagging it here that we may want to remove it from this ticket, it is currently being tracked here:
- #1315 [[Zenhub link](https://app.zenhub.com/workspaces/gc-design-system-6100624a19f4cf000e46e458/issues/gh/cds-snc/design-gc-conception/1315)]

**Comment by adorayi** [comment]
Request for Devs to do a review before closing this ticket.

**Comment by nmakuch** [comment]
Button v2 branch
https://www.figma.com/design/mh2maMG2NBtk41k1O1UGHV/branch/JnOyLhT7sKJ2gZLIlEW1rx/GC-Design-System?node-id=817-4607&p=f&t=sqCpsc4ksyfT7JNg-0

**Comment by nmakuch** [comment]
FR Button branch:
https://www.figma.com/design/o4SguSZdar2CCFzSkWNrmB/branch/KDdeJHASWrKKY7KJ9pvmD4/Syst%C3%A8me-de-design-GC?node-id=8-2054&p=f&t=SK4OHEqrERSXNvyJ-0
